### DISCUSSION TITLE
Team D - Course 1

### DISCUSSION THREAD

**Discussion by 1hk-star** [lablup/contribution-academy-2021#27] [Show and tell]
.

**Comment by youngjun0627** [comment]
210810 이유찬 1주차 Course 1: Backend.AI 사용해보기

Backend.AI : tensorflow & CPU 환경

**1. execute training**

 ![image](https://user-images.githubusercontent.com/57058726/128841020-fbad12b0-cf1f-46fc-9f08-b5c84267e759.png)

**2.  evaluate simple model(Accuracy)**

![image](https://user-images.githubusercontent.com/57058726/128840669-3d1c2335-a645-4a54-9331-f9240ebb2c0b.png)

**3. total allocated resources(small version of nvidia-RTX resource group)**

![image](https://user-images.githubusercontent.com/57058726/128840742-1f6d5a45-227c-482e-89d7-c6c2cf6a40fb.png)


~~시간 남을 때 GPU로도 돌려보고 pytorch로도 함 해보겠습니당~~
***

210811 이유찬 1주차 Course 1: Backend.AI 사용해보기

Backend.AI : pytorch & GPU 환경

1. execute training

![image](https://user-images.githubusercontent.com/57058726/128903746-9d0d7761-219c-44de-8648-cd559a050223.png)

2. evaluate LeNet & simple(DNN) model

LeNet : 0.895,    DNN : 0.886

3. total allocated resources

![image](https://user-images.githubusercontent.com/57058726/128903485-ae70f770-677d-4b35-8a63-1f5235c311bc.png)

source code : https://github.com/youngjun0627/pytorch_fashionMNIST/blob/main/demo.ipynb

**Reply by lizable** [reply]
@1hk-star 
hyper parameter tuning으로 accuracy 높여보시는 것도 어떠신지...?

**Reply by 1hk-star** [reply]
@lizable @youngjun0627 저는 아직 안했습니다.

**Reply by youngjun0627** [reply]
@lizable pytorch로 진행하면서 실험해보도록 하겠습니다.

**Comment by 1hk-star** [comment]
1. execute training
![image](https://user-images.githubusercontent.com/32287560/129017755-918cb05a-b195-44e2-ba08-fd2b3391738a.png)

2. evaluate simple model(Accuracy)
![image](https://user-images.githubusercontent.com/32287560/129017809-b8a516ff-48b7-4416-a637-a824efc48464.png)

3. total allocated resources(small version of nvidia-RTX resource group)
![image](https://user-images.githubusercontent.com/32287560/129017884-26f35302-8c94-47a9-a24e-943ec4662d45.png)

**Comment by hyeyeon-sun** [comment]
1. execute training
![실행시간](https://user-images.githubusercontent.com/39080868/129033207-d0254633-4f7a-456a-9d0a-5b29f8377ca8.JPG)

2. accuracy
![정확도](https://user-images.githubusercontent.com/39080868/129033265-17c8dedc-4916-474f-8b54-e4e0438448db.JPG)

3. total allocated resources:
![total allocated resources](https://user-images.githubusercontent.com/39080868/129033299-eacc9d2a-2b57-409f-a941-8ff7d6493f48.JPG)

**Comment by JHyunB** [comment]
1. execute training
![image](https://user-images.githubusercontent.com/36959292/129037461-da50a0b9-3829-4b03-8d32-ad4f89c74e76.png)

2. accuracy
![image](https://user-images.githubusercontent.com/36959292/129037602-e9a0b353-0c96-400a-b0a2-b0e06a97ca4b.png)

3. total allocated resources 
![image](https://user-images.githubusercontent.com/36959292/129037845-acd92fee-e859-4913-9817-99f096795b47.png)

**Comment by 1hk-star** [comment]
@lizable  완료하였습니다.
